docs/user/filter-and-process/ottl-transform-and-filter/README.md

```diff
@@ -0,0 +1,54 @@
+# Transform and Filter with OTTL
+
+To apply custom transformations and filters to your telemetry data, use the OpenTelemetry Transformation Language (OTTL). You can modify, enrich, and filter logs, metrics, and traces before they're sent to your backend.
+
+## Overview
+
+You define OTTL rules in the spec of your LogPipeline, MetricPipeline, and TracePipeline resources.
+
+With the transform section, you can standardize and enrich your data: Add, rename, or modify fields to conform to a standard schema, add valuable context, or redact sensitive information. For details, see [Transform with OTTL](ottl-transform.md).
+
+With the filter section, you can reduce data volume and cost: Drop unwanted logs, metrics, or traces to lower your ingestion costs and reduce noise in your observability backend. For details, see [Filter with OTTL](ottl-filter.md).
+
+When you define rules, the Telemetry module configures processors in the underlying OpenTelemetry Collector to run your statements.
```

Comment:
```suggestion
When you define rules, the Telemetry module configures processors in the underlying OpenTelemetry Collector to apply your rules.
```

transform processor has conditions and statements, while filter processor has only conditions. So, if we want this sentence to be generic for both features, then let's avoid saying `run your statements`